src/Mappers/TransactionMapper.php

```diff
@@ -93,12 +102,12 @@ public static function map(string $transactionClassName, Response $response): ar
                 $transactionLine = new $transactionLineClassName();
 
                 $transactionLine
-                    ->setType($lineElement->getAttribute('type'))
+                    ->setType(new LineType($lineElement->getAttribute('type')))
                     ->setId($lineElement->getAttribute('id'))
                     ->setDim1(self::getField($transaction, $lineElement, 'dim1'))
                     ->setDim2(self::getField($transaction, $lineElement, 'dim2'))
-                    ->setDebitCredit(self::getField($transaction, $lineElement, 'debitcredit'))
-                    ->setValue(self::getField($transaction, $lineElement, 'value'))
+                    ->setValue(Money::EUR(100 * self::getField($transaction, $lineElement, 'value')))
```

Comment:
Transactions can be in other currencies also. Not sure if we do have to support it yet, but in that case we should at least assert that the transactions currency is `EUR`.

Reply:
Yes, I think we should handle multi-currency when we get there.